.github/workflows/push-notification-jar.yml

```diff
@@ -0,0 +1,44 @@
+name: Upload Common-Utils Jar to Maven
```

Comment:
Just confirming - I remember Cameron sent out some PRs to change up some of the workflows, were those changes affecting this specific type of workflow and did you include the refactoring he did if so? If not related can ignore this comment.

Reply:
Those are planned for post 1.11 release.